internal/controller/postgrescluster/pgadmin_test.go

```diff
@@ -266,7 +266,8 @@ ownerReferences:
 		// Defaults to ClusterIP.
 		assert.Equal(t, service.Spec.Type, corev1.ServiceTypeClusterIP)
 		assert.Assert(t, marshalMatches(service.Spec.Ports, `
-- name: pgadmin
+- appProtocol: tcp
+  name: pgadmin
```

Comment:
❓ pgAdmin is definitely speaking HTTP on this port. Should this be `appProtocol: http` as [registered](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=http) at IANA?

❓ In the future, when pgAdmin is deployed with TLS, will this need to change to `https` to work correctly with Istio?

Reply:
will change to `http` for now.

RE: `https`, it depends. When traffic is identified as `https`, the Istio sidecar(may) not re-encrypt the traffic using mTLS configured in the service mesh. Essentially it'd just forward the traffic to the destination. So as long as `pgadmin` is doing the TLS termination/validation it should be fine.

However, then it could be a headache for whoever's managing the Istio mesh to assert all traffic in the cluster is satisfying certain encryption. But that'd be on them to document. IMO, I'd prefer to keep all traffic between Pods through Istio's mTLS.. though we don't really use `pgadmin` so I'll defer to someone else on that